clang/include/clang/AST/Decl.h

```diff
@@ -1474,6 +1478,16 @@ class VarDecl : public DeclaratorDecl, public Redeclarable<VarDecl> {
     NonParmVarDeclBits.ExceptionVar = EV;
   }
 
+  /// Determine the Ellipsis (...) or not
+  bool isEllipsisVariable() const {
+    return isa<ParmVarDecl>(this) ? false : NonParmVarDeclBits.EllipsisVar;
+  }
+  void setEllipsisVariable(bool EV) {
+    assert(!isa<ParmVarDecl>(this));
+    NonParmVarDeclBits.EllipsisVar = EV;
+  }
```

Comment:
Should we move this to ParmVarDecl?

Reply:
It might impact on the testcases failure , but I will check otherwise it will remain this